python/metatensor-operations/tests/split.py

```diff
@@ -520,7 +516,7 @@ def test_split_errors(self):
             ),
         self.assertEqual(
             str(cm.exception),
-            "each element in grouped_labels must be 'Labels', not <class 'str'>",
```

Comment:
Instead of removing the error message context, the type check should only be done if not in torch script mode (since in torch script mode the types are checked statically)